src/coreclr/hosts/unixcoreconsole/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+project(unixcoreconsole)
+
+# Work around XCode 6.3 SDK issue where __debug is included by some
+# system headers but not present by adding our own include directory
+# with a dummy file called __debug.
+if(APPLE)
+    include_directories(inc)
+endif(APPLE)
+
+include_directories(../unixcoreruncommon)
+
+set(CORECONSOLE_SOURCES 
+    ../unixcoreruncommon/coreruncommon.cpp
+    coreconsole.cpp 
+)
+
+add_executable(coreconsole
+    ${CORECONSOLE_SOURCES}
+)
+
+target_link_libraries(coreconsole 
+    dl
+)
```

Comment:
This needs the same check for freebsd that was just merged in https://github.com/dotnet/coreclr/pull/791